From: David Zafman <dzafman@redhat.com>
Date: Fri, 20 May 2016 18:19:42 -0700
Subject: osd: Fix log messages

Signed-off-by: David Zafman <dzafman@redhat.com>
(cherry picked from commit b40ec3fe890679b4c73778815e1bedd492cb264b)
(cherry picked from commit 120c040a4c5a91b3f368597a8c83703437b32fb7)

Resolves: rhbz#1414613
---
 src/osd/ReplicatedPG.cc | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/src/osd/ReplicatedPG.cc b/src/osd/ReplicatedPG.cc
index 4b879d9..697d8ba 100644
--- a/src/osd/ReplicatedPG.cc
+++ b/src/osd/ReplicatedPG.cc
@@ -8593,7 +8593,7 @@ void ReplicatedPG::submit_log_entries(
   ObcLockManager &&manager,
   boost::optional<std::function<void(void)> > &&on_complete)
 {
-  dout(10) << __func__ << entries << dendl;
+  dout(10) << __func__ << " " << entries << dendl;
   assert(is_primary());
 
   ObjectStore::Transaction t;
@@ -9594,7 +9594,7 @@ void ReplicatedPG::failed_push(pg_shard_t from, const hobject_t &soid)
   assert(recovering.count(soid));
   recovering.erase(soid);
   missing_loc.remove_location(soid, from);
-  dout(0) << "_failed_push " << soid << " from shard " << from
+  dout(0) << __func__ << " " << soid << " from shard " << from
 	  << ", reps on " << missing_loc.get_locations(soid)
 	  << " unfound? " << missing_loc.is_unfound(soid) << dendl;
   finish_recovery_op(soid);  // close out this attempt,
@@ -10239,7 +10239,7 @@ void ReplicatedPG::_clear_recovery_state()
 
 void ReplicatedPG::cancel_pull(const hobject_t &soid)
 {
-  dout(20) << __func__ << ": soid" << dendl;
+  dout(20) << __func__ << ": " << soid << dendl;
   assert(recovering.count(soid));
   ObjectContextRef obc = recovering[soid];
   if (obc) {
